src/main/java/com/marek/onlinebookstore/model/Category.java

```diff
@@ -0,0 +1,31 @@
+package com.marek.onlinebookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
+
```

Comment:
Add `@NoArgsConstructor`, right now it's provided by default, but if in the future you will create another constructor, you will recive an exception without it.